### PR TITLE
DOCSP-41126: write operations

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin-sync/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-41126>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/kotlin-sync/DOCSP-41126-write-operations/write-operations/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/write-operations.txt [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+   :dedent:
+
+.. To learn more about the ``insert_many()`` method, see the :ref:`Insert Documents
+.. <kotlin-sync-write-insert>` guide.
+
+Update One
+----------
+
+The following code shows how to update a single document in a
+collection by creating or renaming a field:
```
Comment (mcmorisi) [inline source/write-operations.txt:95]: Do we also want to mention reassigning the value of a field as an option here?

File: source/write-operations.txt [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+   :dedent:
+
+.. To learn more about the ``update_one()`` method, see the
+.. :ref:`Update Documents <kotlin-sync-write-update>` guide.
+
+Update Multiple
+---------------
+
+The following code shows how to update multiple documents in a
+collection by creating or renaming a field:
```
Comment (mcmorisi) [inline source/write-operations.txt:111]: Same as above

**Review by mcmorisi — APPROVED** [review]
LGTM with a couple of comments, nice work!

File: source/write-operations.txt (hunk 1 above)
Comment (rustagir) [reply source/write-operations.txt]: updated language

**Review by vbabanin — APPROVED** [review]
LGTM!

**Pull request merged**